tests/store/filters/test_analysis_filters.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.orm import Query
+
+from tests.mocks.store_mock import MockStore
+from trailblazer.store.filters.analyses_filters import filter_analyses_by_id
+from trailblazer.store.models import Analysis
+
+
+def test_filter_analyses_by_id(analysis_store: MockStore):
+    """Test return analysis by id when existing."""
+    # GIVEN a store containing analyses
+    existing_analysis: Analysis = analysis_store.get_query(table=Analysis).first()
+
+    # WHEN retrieving an analysis by id
+    analysis: Query = filter_analyses_by_id(
+        analyses=analysis_store.get_query(table=Analysis), analysis_id=existing_analysis.id
+    )
+
+    # ASSERT that the analysis is a query
```

Comment:
I'm not sure what the `ASSERT` keyword adds compared to GWT. Doesn't `THEN` also work here?

Reply:
Fixed